kickerSup/kicker_current.db

```diff
@@ -0,0 +1,96 @@
+record(ai, "$(P)CURR") {
+    field(DESC, "Kicker PSU current")
+    field(EGU, "mA")
+    field(DTYP, "Soft Channel")
+    field(INP,  "$(P)CALIBRATE:CURR CP MS")
+    field(PREC, "5")
+
+    field(HIGH, "$(PSU_MAX_CURR)")
+    field(HSV, "MAJOR")
+
+    field(LOW, "0")
+    field(LSV, "MAJOR")
+
+    info(INTEREST, "HIGH")
+    info(archive, "VAL")
+}
+
+record(calc, "$(P)CALIBRATE:CURR") {
+    field(DESC, "Calibrates the current")
+    field(INPA, "$(DAQMX)CURR:DATA CP MS")
+    field(CALC, "A * $(PSU_MAX_CURR) / 10")
+    field(PREC, "5")
+}
+
+record(subArray, "$(DAQMX)CURR:DATA") {
+    field(DESC, "Reads a single raw current val")
+    field(PINI, "NO")
+    field(DTYP, "Soft Channel")
+    field(PREC, "5")
+    field(INP, "$(DAQMX)CURR:_RAW CP MS")
+    field(NELM, "1")
+    field(INDX, "0")
+    field(FTVL, "DOUBLE")
+}
+
+record(waveform, "$(DAQMX)CURR:_RAW") {
+    field(DESC, "Reads an array of raw current data")
+    
+    $(IFNOTRECSIM) field(SCAN,"I/O Intr")
+    $(IFNOTRECSIM) field(DTYP, "asynFloat64ArrayIn")
```

Comment:
The dual `DTYP` fields here cause the DB unit checker to fail. This will need to be added as an exception to DBUnitChecker before this can be merged. Also applies to the voltage

Reply:
See DBunit checker pull request [here](https://github.com/ISISComputingGroup/EPICS-DbUnitChecker/pull/15)